src/main/resources/templates/pet/giveForAdoption.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+';lbv <!DOCTYPE html>
 <html xmlns:th="https://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
```

Comment:
【typo】
The string [';lbv] before <!DOCTYPE html> seems to be a typo.
Having unnecessary string before <!DOCTYPE html> may cause an error.